terraform/main.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 0.13.1"
```

Comment:
While we're on it, can we upgrade to a more recent TF version?

Reply:
Yep. This was actually deployed using 1.3.something on my workstation, so 0.13.1 is no longer accurate